x-pack/plugins/observability/public/pages/slo_details/components/header_title.tsx

```diff
@@ -22,14 +22,14 @@ export function HeaderTitle(props: Props) {
     return <EuiLoadingSpinner data-test-subj="loadingTitle" />;
   }
 
+  if (!slo) {
+    return null;
+  }
```

Comment:
📣  This is just to please typescript, but this is an impossible scenario: if the slo is not loading, it is there. Otherwise, we show the 404 page higher up in the component tree.